dockers/docker-ptf/Dockerfile.j2

```diff
@@ -100,4 +100,4 @@ COPY ["supervisord.conf", "sshd.conf", "ptf_nn_agent.conf", "/etc/supervisor/con
 
```

Comment:
I can rename conf.d/supervisord.conf file to something else. But I think it's good enough name to show that we're changing supervisord configuration in the file.